themes/vocabulary_theme/templates/cc-search-guide.html

```diff
@@ -4,15 +4,15 @@
 
 {% block body %}
 <div class="cc-search-guide">
-  <header class="header">
+  <header class="header" role="banner">
```

Comment:
the `header` already has its semantic role, and shouldn't need the `role` of `banner` here as far as I know.